synth.py

```diff
@@ -25,40 +25,20 @@
 versions = ['v1']
 for version in versions:
   library = gapic.typescript_library(
-    '@google-cloud/monitoring-dashboard',
+    'monitoring-dashboard',
     generator_args={
+      'package-name': '@google-cloud/monitoring-dashboards'
     },
     proto_path='/google/monitoring/dashboard/v1',
     version=version)
-s.copy(library, excludes=['README.md'])
+s.copy(library, excludes=['README.md', 'package.json'])
 
 # Copy common templates
 common_templates = gcp.CommonTemplates()
 templates = common_templates.node_library(source_location='build/src')
-s.copy(templates, excludes=[])
+s.copy(templates, excludes=['.github/release-please.yml'])
 
 # Node.js specific cleanup
-subprocess.run(['rm', '-rf', './system-test/fixtures'])
 subprocess.run(['npm', 'install'])
 subprocess.run(['npm', 'run', 'fix'])
```

Comment:
need to add `subprocess.run(['npx', 'compileProtos', 'src'])`, otherwise the protos JSONs won't be recompiled in case of the proto file update.